language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/performance/CompletionOpenDocPerformanceTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.langserver.performance;
+
+import com.google.gson.JsonObject;
+import org.ballerinalang.langserver.util.FileUtils;
+import org.ballerinalang.langserver.util.TestUtil;
+import org.eclipse.lsp4j.jsonrpc.Endpoint;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+
+/**
+ * Test performance of openDocument in completions feature in language server.
+ */
+public class CompletionOpenDocPerformanceTest extends CompletionPerformanceTest {
```

Comment:
I assume this test is to measure performance of `openDocument` call? If so, we should rename this to imply that. Or, we can include the `openDocument` performance tests in other tests (like completion) because we are going to perform the open document step anyway.